frontend/src/metabase/embedding/embedding-hub/components/EmbeddingHub.tsx

```diff
@@ -95,6 +119,9 @@ export const EmbeddingHub = () => {
         opened={openedModal?.type === "xray-dashboard"}
         onClose={closeModal}
       />
+      {openedModal?.type === "user-strategy" && (
+        <PLUGIN_TENANTS.EditUserStrategyModal onClose={closeModal} />
+      )}
```

Comment:
In the design, the modal opens within the setup guide. It doesn't navigate to the people page. 

<img width="1258" height="734" alt="CleanShot 2568-12-18 at 21 49 19@2x" src="https://github.com/user-attachments/assets/a23de33f-8b6d-477f-b626-9149ba3e4a16" />